### DISCUSSION TITLE
shouldn't ask for curb cuts on alleyways

### DISCUSSION THREAD

**Discussion by anarcat** [streetcomplete/StreetComplete#5283] [Feedback]
So I must first say that I love street complete. It's one of the best things to ever happen to OSM and I think it's doing absolutely fantastic work to improve the map. I've been using it for a while, and it's a blast, thanks!

That said, there's been a change recently in my city that makes it slightly less fun to use. It is asking for curb heights where the sidewalk crosses an alleyway and, in my city (Montreal), I can't actually think of any situation where this would occur. When a sidewalk "crosses" a street, of course, there's a curb and a curb cut, and it can be done properly or not and that question is valid and I absolutely want to document this for fellows (like me!) with "accessibility needs".

But in alleyways? Not really! Here, the sidewalk is a solid block of concrete that runs around the block and the alleyway basically stops flush to it and doesn't disturb its path. So the question doesn't make much sense.

And because of this mismatch, we end up being asked *five* questions for every sidewalk crossing an alleyway:

1. what's the height of this curb? (first side of alleyway)
2. what's the height of this curb? (second side of alleyway)
3. is there tactile pavement on this curb here? (first side)
4. is there tactile pavement on this curb here? (second side!)
5. does this crosswalk have tactile paving on both ends?

here's a few screenshots showing this behavior:

![Screenshot_20231001-002730](https://github.com/streetcomplete/StreetComplete/assets/796623/63a3947a-8f4a-4f51-8025-487041b1df62)
![Screenshot_20231001-002752](https://github.com/streetcomplete/StreetComplete/assets/796623/c2f95cad-bde7-40e1-b488-e746ea60a41b)
![Screenshot_20231001-002801](https://github.com/streetcomplete/StreetComplete/assets/796623/0ff2252f-e668-4fbe-ad3c-f41728290c7d)

That particular crossing is https://www.openstreetmap.org/node/11169551303 but it's not specific to that one.

I suspect this is not a new bug in StreetComplete, but rather a change due to the way sidewalks are mapped in Montreal (they are now, they were not marked separately before). So maybe this is a bug in the base map as well...

Anyway, just thought you should know and wondering if anyone else has seen stuff like this...

**Comment by westnordost** [comment]
Well, StreetComplete necessarily works with what is mapped already and can't judge if the underlying data is questionable.

The two quests for the curb cuts are shown because the section that crosses the alleyway is mapped as a `footway=crossing` inbetween two `footway=sidewalk`: https://www.openstreetmap.org/way/1205209674

The quest that asks for whether the crossing has tactile paving is shown for the node because that node is tagged as a crossing: https://www.openstreetmap.org/node/11169551303 - many people in OSM have the opinion that driveways that cross the sidewalk (without any curb cuts) should not be tagged with `highway=crossing` at all.

So, I'd say that at least https://www.openstreetmap.org/way/1205209674 is wrong, as the footway does not cross the alleyway, the alleyway crosses the footway.

**Comment by BalooUriza** [comment]
At least in the US, we're all over the place with alleyways.  In my city
specifically, we very reliably have high curbs on alleys built before 1970
and almost never on alleys built after 2000 (despite federal laws banning
existing and new installations of high curbs at pedestrian crossings in the
1980s).
